.gitignore

```diff
@@ -4,3 +4,4 @@
 coverage
 dist
 node_modules
+/docs/_API-body.md
```

Comment:
This is an intermediary file that shouldn't be checked-in.  It can be left in the filesystem when using the `docs-watch` package.json script, though.